.expeditor/config.yml

```diff
@@ -151,14 +151,16 @@ subscriptions:
   - workload: buildkite_hab_build_group_published:{{agent_id}}:*
     actions:
       - built_in:promote_habitat_packages
-      - bash:.expeditor/upload-files.sh:
+      - trigger_pipeline:post-promote:
+          post_commit: true
+  - workload: buildkite_build_passed:{{agent_id}}:post-promote:*
+    actions:
+      - purge_packages_chef_io_fastly:dev/automate/latest:
```

Comment:
I think we want to purge the cache before deploying to dev to make sure that dev picks up the latest manifest.